content/blog/azure-native-v3/index.md

```diff
@@ -0,0 +1,602 @@
+---
+title: "Azure Native V3: A Leaner, Faster SDK for Microsoft Azure"
+date: 2025-04-03
+draft: false
+meta_desc: Azure Native V3 delivers 75% smaller SDK size, updated API versions, and flexible options for working with explicit API versions in Microsoft Azure.
+meta_image: meta.png
+authors:
+    - thomas-kappler
+    - meagan-cojocar
+
+tags:
+    - azure-native
+    - azure
+    - features
+    - releases
+
+social:
+    twitter:
+    linkedin: 
+---
+
+Today we're excited to announce the release of Pulumi Azure Native V3, a major update to our Azure Native provider that improves developer experience through significant SDK size reduction, up-to-date default API versions, and a flexible approach to managing explicit API versions.
+
+[Pulumi Azure Native](/registry/packages/azure-native/) is our native provider for Microsoft Azure, offering direct access to the  [Azure Resource Manager API](https://learn.microsoft.com/en-us/rest/api/resources/). Unlike traditional providers that abstract away cloud APIs, Azure Native gives you more coverage of Azure resources, properties, and up-to-date features by directly mapping to Azure's native REST API. This allows you to provision and manage any Azure resource with Pulumi in your favorite programming language while taking advantage of all the latest Azure capabilities as soon as they're released.
```

Comment:
Can we reiterate and retell the original value prop? I think we have like more than 2x resource/property coverage than Terraform at this point. We shouldn’t bury the lede. Let’s be loud and proud about this - we are hands down the best IaC solution for Azure and we just removed the only possible downside (perf/size).

Eg see charts from https://www.pulumi.com/blog/announcing-nextgen-azure-provider/. What would modern day revisions of them look like?